.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -111,9 +111,9 @@ def build_spec_backcompat_suite():
 def build_spec_celery_k8s_suite():
     tox_env_suffixes = [
         "-default",
-        "-markusercodedeployment",
```

Comment:
was this here to test non-subchart version?

Reply:
I took this out b/c it wasn't substantively different than default anymore - both now use user-code-deployments